distributed/comm/tests/test_ucx_config.py

```diff
@@ -95,23 +95,26 @@ async def test_ucx_config(cleanup):
 
 
 @pytest.mark.flaky(
-    reruns=10, reruns_delay=5, condition=ucp.get_ucx_version() < (1, 11, 0)
```

Comment:
Do we guarantee having 1.11.0+ at this point? Do we document what versions we support anywhere?

Reply:
Got it. Might be worth thinking about how to document UCX compatibility in Dask at some point

Reply:
That is actually left to UCX-Py, not Dask, so it's gonna be whatever UCX-Py supports, therefore I don't think it makes sense to document UCX as it's an indirect dependency. This will be more evident soon when we drop support for UCX < 1.11.1 and then there will be no more checks for UCX version in code, I have changes prepared but am still in the process of testing multi-GPU to ensure nothing breaks.